src/client/components/Auth/AuthenticatedRoute.js

```diff
@@ -1,11 +1,11 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { useAuthentication } from '../../hooks/useAuthentication';
 import { Redirect, Route } from 'react-router-dom';
 import Loader from '../Loader';
+import { useFirebase } from '../../firebase/FirebaseContext';
 
 function AuthenticatedRoute({ children, ...rest }) {
-  const { isAuthenticated, isLoading } = useAuthentication();
+  const { isAuthenticated, isLoading } = useFirebase();
```

Comment:
Great.
We need to replace useAuthentication with useFirebase everywhere. Not just here